src/lib/utils/__tests__/track-tests.ts

```diff
@@ -1,9 +0,0 @@
-import { Schema } from "../track"
-
-describe("analytics constants", () => {
-  it("action names all start with a lowercase", () => {
-    // grab string values from enum
-    const values = Object.keys(Schema.ActionNames).map(k => Schema.ActionNames[k as any])
-    values.map(name => expect(name[0] === name[0].toLowerCase()).toBeTruthy())
-  })
-})
```

Comment:
this test broke but it's only enforcing an old convention that we broke here to maintain compatibility with eigen's analytics schema.

Reply:
(so i just deleted it)